modules/openapi-generator/src/test/resources/3_0/spring/petstore-with-fake-endpoints-models-for-testing.yaml

```diff
@@ -26,7 +26,7 @@ paths:
       requestBody:
         $ref: "#/components/requestBodies/Pet"
       responses:
-        "200":
+        "201":
```

Comment:
Is there a reason to make any changes in this file? If not, could you revert them please?

Reply:
Just wanted to use correct statuses for POST/PUT success responses. Gonna revert then.